docs/examples/Comparing_EHM_JPDA_example.py

```diff
@@ -0,0 +1,435 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+============================================================================
+Comparing Efficient Hypothesis Management (EHM) with probability associators
+============================================================================
+"""
+
+# In this example, we compare the performances between efficient hypothesis management
+# (EHM) and standard joint probabilistic data association.
+# The problem we face when dealing with multi-target tracking is the potential
+# association of measurements to predictions, which can be ambiguous and
+# that could lead to a combinatorial explosion. To reduce the computational
+# cost of the operations, a number of algorithms have been developed to
+# match measurements and predictions to tracks. One of this methods is the
+# efficient hypothesis management, explained in detail in [1]_, [2]_ and under
+# patent in [3]_; this algorithm improves the joint probability data association,
+# which is a brute force approach, with improved capability of hypothesis matching
+# and rejection with, significantly, cost reduction.
+#
+# As it stands, there is not a Stone Soup implementation yet, however we
+# base these comparisons on the publicly available, under patent license agreement,
+# Python package PyEHM developed by ` Dr. Lyudmil Vladimirov `_.
+# .. _Lyudmil Vladimirov: https://github.com/sglvladi/pyehm
```

Comment:
So this needs to be reworded to highlight that a Stone Soup implementation does exist, but as a plugin provided by PyEHM (under patent license agreement); rather than being part of the main Stone Soup package.